src/components/Card.css

```diff
@@ -1,6 +1,11 @@
 .Card {
   display: table;
   word-break: break-word;
+  cursor: pointer;
```

Comment:
Maybe would be better to control the cursor pointer on the component, if `props.onClick` exists then cursor pointer otherwise inherit